index.bs

```diff
@@ -1026,26 +1026,66 @@ Issue: Define how this works.
 <dfn>RemoteReference</dfn> represents a remote reference to an exising ECMAScript object in
 [=object id map=].
 
-A [=BiDi session=] has an <dfn>object id map</dfn>. This is a weak map
-from objects to their corresponding id.
+A [=BiDi session=] has:
+ * <dfn>object id map</dfn>, a weak map from objects to their corresponding id, unique for each Sandbox.
+ * <dfn>dom id map</dfn>, a weak map from [=platform object=] to their corresponding id and shared among Sandboxes.
```

Comment:
This would mean everything that is defined using Web IDL, per https://webidl.spec.whatwg.org/#dfn-platform-object

Is that really what is intended?

Reply:
Yes, any Web IDL object CAN be put there. But serialization will define, if the y WILL. I'd start with adding only Nodes to the map. This PR is intended to:
1. Proto an discuss the deserialization approach discussed on the W3C meeting.
2. Figure out the difference between ECMAScript wrappers and WebIDL on the spec level.

Reply:
I see, that makes sense, we can make it such that this weak map can map any platform object to a stable ID, but only but `Node`s in the map for now. How to deal with `Window` then becomes a question of whether we put it in the map.

Reply:
I wonder if shared array buffers are another special case.

Reply:
I made the reference independent of the platform objects, and rely only on `SandboxProxy`. It should make the approach scalable enough, assuming the `SandboxProxy` can be defined for **shared array buffers**.

Reply:
@jgraham do you mean that we should have a stable id for shared array buffers between a "main" context and sandboxes, or for the "same" shared array buffer in a window and worker context?

Reply:
Discussed in a chat with James:

@jgraham 

> "I don't know" is pretty much my answer there. It just occured to me that SAB might be another case where we have the same object in multiple contexts. For sharing DOM nodes between Window and Sandbox globals we have pretty strong use cases; I don't know if we have corresponding requirements for SAB or if it's always OK to treat them as seperate objects per realm.

@foolip 

> To get the "same" SAB in two places would require `postMessage` I guess. Unlike with `Node`s, it's not already there in the other sandbox. So this feels to me like if one really needs to keep track of which SABs are the same, one can `postMessage` a random ID together with it.
>
> Which is all to say I think it would be nice to stick to just platform objects unless we have strong use cases for other kinds of objects.

@jgraham 

> Yeah, I think I buy that argument. I guess the worst case here is we later serialize some kind of `arrayBufferId` that can then be used to look up the same SAB. So if there are use cases that come up in the futute we can probably deal with them without doing anything special now.

Reply:
@jgraham do I use the `SandboxProxy` in the way it was intended to?

Reply:
Done. Restricted to the platform objects.